Extension/i18n/deu/src/Debugger/ParsedEnvironmentFile.i18n.json

```diff
@@ -4,5 +4,5 @@
  *--------------------------------------------------------------------------------------------*/
 // Do not edit this file. It is machine generated.
 {
-	"ignoring.lines.in.envfile": "Nicht analysierbare Zeilen in {0} {1} werden ignoriert: "
-}
+	"ignoring.lines.in.envfile": "Nicht analysierbare Zeilen in {0} {1}werden ignoriert: "
```

Comment:
add back the space?

Reply:
I think something's wrong with this one in general though. It ends with a `:` and a space like it expects the `{1}` at the end.

Reply:
I think that is as expected.  All the other strings of this ID are formatted the same way (": ").  Code outputs one or more lines of text after this label.